source/uwp/Renderer/lib/Util.cpp

```diff
@@ -699,4 +699,10 @@ HRESULT ProjectedElementTypeToHString(ABI::AdaptiveCards::Uwp::ElementType proje
 {
     CardElementType sharedElementType = static_cast<CardElementType>(projectedElementType);
     return UTF8ToHString(CardElementTypeToString(sharedElementType), result);
-}
\ No newline at end of file
+}
+
+bool IsWarning(int FailureStatusCode)
+{
+    int mask = 0xFF000000;
```

Comment:
>0xFF000000 [](start = 15, length = 10)

This should be 0xF0000000

Reply:
Function is no longer needed

---
In reply to: [151202047](https://github.com/Microsoft/AdaptiveCards/pull/950#discussion_r151202047) [](ancestors = 151202047)